v1/lists.js

```diff
@@ -82,6 +94,8 @@ class ReadingLists {
                 action: 'query',
                 list: 'readinglistentries',
                 rlelists: req.params.id,
+                rlesort: this.getSortParameters(req.query.sort).sort,
```

Comment:
Nit: this is the same as `rlesort: req.query.sort`. If you do want to call the function nevertheless and use its return value (because the logic inside it might change), better to do it before returning the request and store the result in a variable so that it's not called twice with the same param as the return value can be reused on the line below. That not only saves time, but also reduces the memory footprint.